hypha/apply/projects/templatetags/approval_tools.py

```diff
@@ -40,3 +41,8 @@ def user_can_final_approve_project(project, user):
 @register.simple_tag
 def user_can_edit_project(project, user):
     return project.editable_by(user)
+
+
+@register.simple_tag
+def generate_fund_page_admin_url(page):
```

Comment:
```suggestion
def fund_page_admin_url(page):
```

Reason: the function is not generating the URL.

Also, curious why not using `{% url "funds_fundtype_modeladmin_edit", page.id %}`, another recommendation here is to use `fund` as function arg instead of a `page`. This function is not capable of generating a fund URL from any `page` object.

Reply:
Yes, adding it directly to the template is better. The `page` is denoting the fund here, we have page and round two fields in the submission table and both have relation to the Page model only.